Makefile

```diff
@@ -209,6 +209,7 @@ docker-ci-integration:
 .PHONY: docker-build-collection
 docker-build-collection:
 	docker build -f cmd/Dockerfile  --build-arg TARGET=collection --build-arg COMMIT=$(COMMIT)  --build-arg VERSION=$(IMAGE_TAG) --build-arg GOARCH=$(GOARCH) --target production \
+		--label "git_commit=${COMMIT}" --label "git_tag=${IMAGE_TAG}" \
```

Comment:
What is label for? The `--build-arg` already contains COMMIT and IMAGE_TAG?

Reply:
`--build-arg` passes the value to a `Dockerfile` for building. `--label` adds a metadata to an image which can be later seen by inspecting the image itself

Reply:
I see. Good point. Thanks